remote.tf

```diff
@@ -15,7 +15,7 @@ data "terraform_remote_state" "b2-bucket" {
   config = {
     organization = "gtis"
     workspaces = {
-      name = "b2-${local.app_name_and_env}"
+      name = "b2-${var.app_name}-${var.app_env}"
```

Comment:
Why are these reverting back to variables? Would you ever want `var.app_env` to be different than `data.terraform_remote_state.common.outputs.app_env`?

Reply:
We destroyed `local.app_name_and_env`.  If someone else uses the Terraform module `terraform-aws-tfc-backup-to-b2`, they wouldn't want to be forced to use our `terraform_remote_state`.  They might prefer to set variables in Terraform Cloud.  Consider this code to be our example of "how we use `terraform-aws-tfc-backup-to-b2`.

Reply:
Then why not give better flexibility and do: